### PR TITLE
add automate exception message for invalid enum value as api request parameter

### PR DESCRIPTION
**Pull request by EsmaeilChitgar**
### Describe your changes:

Fixes <issue-number>

When enum value is incorrect in api call, the invalidEnumValue()method generates the exactly message as we expect. for example the Include.java in databaseServiceResource.java list() api. There is a unit test example that shows how to use invalidEnumValue() method instead of traditional way i.e. writing plain string.

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DatabaseServiceResourceTest.java [hunk 1]
```diff
@@ -15,18 +15,14 @@
 
 import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
 import static javax.ws.rs.core.Response.Status.OK;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (harshach) [inline openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DatabaseServiceResourceTest.java:18]: @EsmaeilChitgar please avoid wild-card imports

File: openmetadata-service/src/main/java/org/openmetadata/service/exception/CatalogExceptionMessage.java [hunk 2]
```diff
@@ -14,16 +14,16 @@
...
 import java.util.List;
 import java.util.UUID;
+import java.util.stream.Collectors;
 import org.openmetadata.schema.api.events.CreateEventSubscription;
 import org.openmetadata.schema.api.teams.CreateTeam.TeamType;
 import org.openmetadata.schema.entity.teams.Team;
 import org.openmetadata.schema.type.ChangeEvent;
 import org.openmetadata.schema.type.MetadataOperation;
 import org.openmetadata.schema.type.TagLabel;
-import org.openmetadata.schema.type.TaskType;
```
Comment (harshach) [inline openmetadata-service/src/main/java/org/openmetadata/service/exception/CatalogExceptionMessage.java:25]: removing these from imports throwing build errors ```[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project openmetadata-service: Compilation failure: Compilation failure: 
[ERROR] /Users/harsha/Code/OpenMetadata/openmetadata-service/src/main/java/org/openmetadata/service/exception/CatalogExceptionMessage.java:[230,41] cannot find symbol
[ERROR]   symbol:   class EntityLink
[ERROR]   location: class org.openmetadata.service.exception.CatalogExceptionMessage
[ERROR] /Users/harsha/Code/OpenMetadata/openmetadata-service/src/main/java/org/openmetadata/service/exception/CatalogExceptionMessage.java:[230,64] cannot find symbol
[ERROR]   symbol:   class TaskType
[ERROR]   location: class org.openmetadata.service.exception.CatalogExceptionMessage
[ERROR] /Users/harsha/Code/OpenMetadata/openmetadata-service/src/main/java/org/openmetadata/service/exception/CatalogExceptionMessage.java:[234,62] cannot find symbol
[ERROR]   symbol:   class TaskType
[ERROR]   location: class org.openmetadata.service.exception.CatalogExceptionMessage
[ERROR] -> [Help 1]
```

**Comment by harshach** [comment]
@EsmaeilChitgar left comments on the PR. You have build failures in the PR that's causing tests to fail.
You can also run the tests locally 
```
mvn clean package 
```

**Comment by harshach** [comment]
@EsmaeilChitgar left comments, can you please take a look

File: openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DatabaseServiceResourceTest.java (hunk 1 above)
Comment (EsmaeilChitgar) [reply openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DatabaseServiceResourceTest.java:18]: Will avoid wild-card imports. Thanks.

Comment (EsmaeilChitgar) [reply openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DatabaseServiceResourceTest.java:18]: Thanks for your feedback. I'll apply it as soon as possible.

**Review by EsmaeilChitgar — COMMENTED** [review]
@harshach 
Fix reported issues.

**Review by EsmaeilChitgar — COMMENTED** [review]
done

**Comment by harshach** [comment]
Thanks @EsmaeilChitgar will merge in once all the checks passes

**Pull request merged**